libkbfs/folder_branch_ops.go

```diff
@@ -4663,6 +4663,10 @@ func (fbo *folderBranchOps) notifyOneOpLocked(ctx context.Context,
 			return nil
 		}
 	}
+	// Cancel any block prefetches for unreferenced blocks.
+	for _, ptr := range op.Unrefs() {
```

Comment:
There's a few `return nil` cases above this -- we should probably move this above the big switch.